server/views/pages/applications/change/partials/add-emergency-pin-phone-credit.njk

```diff
@@ -14,7 +14,7 @@
     text: "Use only multiples of £1, with a limit of £5"
   },
   inputmode: "numeric",
-  value: sessionData.amount or application.requests[0].amount,
+  value: amount or sessionData.amount or application.requests[0].amount,
```

Comment:
It's better to do this logic in the routes file and just pass `amount` to the template (same for `reason`), that way the template is cleaner 🙂